tests/sparseml/onnx/utils/test_models.py

```diff
@@ -43,35 +46,27 @@
 
 @pytest.fixture(
     params=[
-        (
-            {
-                "domain": "cv",
-                "sub_domain": "classification",
-                "architecture": "mobilenet_v1",
-                "sub_architecture": "1.0",
-                "framework": "pytorch",
-                "repo": "sparseml",
-                "dataset": "imagenet",
-                "training_scheme": None,
-                "sparse_name": "base",
-                "sparse_category": "none",
-                "sparse_target": None,
-            }
-        ),
+        "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/base-none",
     ]
 )
 def onnx_models_with_data(request) -> OnnxModelDataFixture:
-    model_args = request.param
-    model = Zoo.load_model(**model_args)
-    model_path = model.onnx_file.downloaded_path()
-    data_paths = [data_file.downloaded_path() for data_file in model.data.values()]
+    # clear cache
```

Comment:
deleting the entire cache again?

Reply:
Proposal for fixing the cache conundrum proposed above.